rest/data/interface.go

```diff
@@ -159,6 +162,15 @@ type TestResultsFilterAndSortOptions struct {
 	BaseResults  *TestResultsOptions
 }
 
+// TestSampleOptions specifies the tasks to get the sample for
+// regexes to filter the test names by.
+type TestSampleOptions struct {
+	TaskID          string
```

Comment:
The route should accept a list of task ids.

Reply:
My thinking here was that they'd provide a list of these TestSampleOptions [here](https://github.com/ybrill/cedar/blob/302f05143eb34e5ad2a48995a8434a00aa9e00dd/rest/test_results_routes.go#L177).
Alternatively, I could wrap the list in an object (like I do [on the DB side](https://github.com/ybrill/cedar/blob/302f05143eb34e5ad2a48995a8434a00aa9e00dd/model/test_results.go#L589-L593))

Reply:
I guess that is fine. I think they were expecting a many-to-many mapping of task ids to regular expressions. In this case they would have to duplicate the regex list for each task ID.

Reply:
Ah, that would make more sense.
Let me clarify that.

Reply:
I've changed the interface to be as you suggested: a list of tasks and a single list of filters.